src/workerd/api/streams/common.h

```diff
@@ -29,7 +29,8 @@ class TransformStreamDefaultController;
 
 enum class StreamEncoding {
   IDENTITY,
-  GZIP
+  GZIP,
+  BROTLI
```

Comment:
Doesn't adding this enum value, without handling it in the streams implementation, cause some switch statements to fail to compile because they don't handle it?

Reply:
As far as I can see adding another value to StreamEncoding should be fine, there's no switches checking the value of StreamEncoding and while some branches for EncodedAsyncInputStream/EncodedAsyncOutputStream expect having identity or gzip encoding the Workers Preview-related stream only interacts with kj-based streams.